src/validator.cpp

```diff
@@ -390,13 +390,8 @@ void Validator::validateModel(const ModelPtr &model, std::string filename)
         addError(err);
     }
 
-
     // If we don't have a filename or working directory we can't check imports 
-    bool checkImports = false;
-    if (filename != "") {
-        mPimpl->validateImportSources(model, filename);
-        checkImports = true;
-    }
+    if (filename != "") mPimpl->validateImportSources(model, filename);
```

Comment:
that's a no to this sort of thing. our coverage testing requires one statement per line.

Reply:
OK, ta, will change asap